prometheus_client/bridge/newrelic.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import
+
+import newrelic.agent
+
+from .. import core
+
+
+@newrelic.agent.data_source_factory(name='Prometheus')
+class PrometheusDataSource(object):
+    def __init__(self, settings, environ):
+        self.registry = settings.get('registry', core.REGISTRY)
```

Comment:
It's possible to use it when using `register_data_source`, but the docs do suggest to use string-value settings for everything (as you can only set string values in the .ini file). I think I need to be able to use a custom registry in the tests, but I'll see if I can find another way to do that.